### ISSUE TITLE
Template repos for samples

### ISSUE THREAD

**Issue by Wavesonics** [badoo/Decompose#170]
Having the samples be inside this repo means they aren't fully representative of what they would look like as separate projects.

Their gradle files reference the modules declared in this repo and such.

I would love to see them as separate, completely independent repositories so a new person could look at them and see exactly how the build files are supposed to work.

Projects like [this](https://github.com/theapache64/create-compose-app) are trying to simplify the setup process, but still leave a lot to be desired. I think setting these samples up as stand alone repos would go a long way to simplifying the first time creation process.

**Comment by arkivanov** [comment]
Yeah, I already received such a feedback before. The issue here is that samples are also used for validation and testing. I'm running samples to also manually verify some things. And they are built on CIvon every PR, which is also useful for me. I'm considering to convert samples to standalone projects, but add Decompose deps as [Composite Builds](https://docs.gradle.org/current/userguide/composite_builds.html).
